docs/samples/python/transformers_recognizer/transformer_recognizer.md

```diff
@@ -0,0 +1,41 @@
+# Run Presidio With Transformers Models
+
+In the following example you will see how extract PII entities using transformers models.
+When initializing the `TransformersRecognizer`, choose from the following options:
+1. A string referencing an uploaded model to HuggingFace. Use this url to access all TokenClassification models - https://huggingface.co/models?pipeline_tag=token-classification&sort=downloads
```

Comment:
This would also work for a custom-built model, right? In this case a user would need to pass the path to the stored model